src/js/charts/line.js

```diff
@@ -124,6 +134,7 @@
   }
 
   function mg_default_color_for_path(this_path, line_id) {
+    this_path.classed('mg-line-color', true);
     this_path.classed('mg-line' + (line_id) + '-color', true);
```

Comment:
Now that I look at it, could you not write this as:

```js
this_path.classed('mg-line-color', true)
              .classed('mg-line' + (line_id) + '-color', true);
```

?

Reply:
yeah that looks better, I'll put it that way :+1: